test-kitchen.gemspec

```diff
@@ -30,6 +30,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "safe_yaml",       "~> 1.0"
   gem.add_dependency "thor",            "~> 0.18"
   gem.add_dependency "mixlib-install",  ">= 1.2", "< 3.0"
+  gem.add_dependency "chef",            "~> 12.17"
```

Comment:
Sorry for the delay, been a long Christmas break!

I thought this might be an issue :/ unfortunately I can't see any robust way of encrypting the data bags without using the Chef codebase to do so.

Have you got any examples of plugins that augment the default sandbox provisioning process? I think this is a huge usability boost when writing kitchen tests, and wouldn't mind publishing it as a separate plugin if need be.